OloEngine/tests/Rendering/PropertyTests/RenderPropertyTest.cpp

```
@@ -0,0 +1,265 @@
+        ::glCreateTextures(GL_TEXTURE_2D, 1, &tex);
+        ::glTextureStorage2D(tex, 1, GL_RGBA32F, static_cast<GLsizei>(width), static_cast<GLsizei>(height));
+        ::glTextureSubImage2D(tex, 0, 0, 0, static_cast<GLsizei>(width), static_cast<GLsizei>(height),
+                              GL_RGBA, GL_FLOAT, pixels);
+        ::glTextureParameteri(tex, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
+        ::glTextureParameteri(tex, GL_TEXTURE_MAG_FILTER, GL_LINEAR);
+        ::glTextureParameteri(tex, GL_TEXTURE_WRAP_S, GL_CLAMP_TO_EDGE);
+        ::glTextureParameteri(tex, GL_TEXTURE_WRAP_T, GL_CLAMP_TO_EDGE);
+        return static_cast<u32>(tex);
+    }
+
+    u32 CreateUniformFloatTexture2D(u32 width, u32 height, f32 r, f32 g, f32 b, f32 a)
```

Cppcheck (2.20.0)

[style] The function 'CreateUniformFloatTexture2D' is never used.

(unusedFunction)